css/styles.css

```diff
@@ -22,30 +18,291 @@ a:hover {
     color: hotpink;
 }
 
-div, nav, li, a, ul {
-    display: inline-block;
+ul {
+    display: flex;
+    list-style-type: none;
+    align-items: center;
 }
 
 img {
     width: 25px;
     height: 25px;
 }
 
-.border-right {
-    border-right-style: solid;
-    border-right-color: #eee;
-    border-right-width: 2px;
+header {
+    background-color: #111;
+    color: #eee;
+    width: 100%;
+    font-family: "serif";
+    display: flex;
+    justify-content: space-between;
+    padding: 1em 0;
+    align-items: center;
+}
+
+nav {
+    display: flex;
+    align-items: center;
+}
+
+/* Font Colors */
+.phantomBlue {
+    color: #0d2e57;
+}
+
+.blue500 {
+    color: #3954d9
+}
+
+.grey61 {
+    color: #616161;
+}
+
+.black12 {
+    color: #121212
+}
+
+/* Font Sizes */
+.size15 {
+    font-size: 15px;
+}
+
+.size22 {
+    font-size: 22px;
+}
+
+.size33 {
+    font-size: 33px;
+}
+
+.size38 {
+    font-size: 43px;
+}
+
+.size43 {
+    font-size: 43px;
+}
+
+.size52 {
+    font-size: 52px;
+}
+
+/* Font Families */
+.montserrat {
+    font-family: 'Montserrat', sans-serif;
+}
+
+.poppins {
+    font-family: 'Poppins', sans-serif;
+}
+
+.bebasNeue {
+    font-family: 'Bebas Neue', sans-serif;
+}
+
+/*Font Weights */
+.light {
+    font-weight: 300;
+}
+
+.regular {
+    font-weight: 400;
+}
+
+.medium {
+    font-weight: 500;
+}
+
+.semibold {
+    font-weight: 600;
+}
+
+.bold {
+    font-weight: 700;
+}
+
+.black {
+    font-weight: 900;
+}
+
+.paddingZero {
+    padding: 0
 }
 
-#ecom-tools {
+.paddingTop2 {
+    padding-top: 2em;
+}
+
+
+#ecomTools {
     float: right;
 }
 
-#empty-hamburger {
+#emptyHamburger {
     width: 30px;
     color: #1110
 }
 
-#portpro {
-    font-size: 20px;
+.lorem {
+    font-variant: normal;
+    text-transform: none;
+    font-family: 'Montserrat', sans-serif;
+    font-weight: 300;
+    color: #8a8a8a;
+}
+
+.icons {
+    display: flex;
+}
+
+#sprinter {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-size: xx-large;
```

Comment:
it's better to have numeric values

Reply:
Addressed in MR #4 : https://github.com/simonn01/frontend-chapter-sport-page/pull/4/commits/c8f02f2d88556796c1f7ccfbfea7d6fc61dd5f9a